src/components/DpButton/DpButton.vue

```diff
@@ -263,7 +263,8 @@ const allColorClasses = {
       active:bg-interactive-subtle-active active:text-interactive-active `,
     // solid: classes that only apply to "solid" button color variant.
     solid: `
-      bg-interactive text-on-dark
+      bg-interactive
+      text-on-dark hover:text-on-dark hover:no-underline
       hover:bg-interactive-hover
```

Comment:
my idea was to have one line for each state: default state, hover, focus, and so on.

```suggestion
      bg-interactive text-on-dark
      hover:bg-interactive-hover hover:text-on-dark hover:no-underline
```